extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -147,11 +148,26 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				}
 			}));
 
+		this._sqlVmPageBlobInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				text: constants.DATABASE_BACKUP_SQL_VM_PAGE_BLOB_INFO,
+				style: 'information',
+				width: WIZARD_INPUT_COMPONENT_WIDTH,
+				CSSStyles: { ...styles.BODY_CSS, 'display': 'none' },
+				links: [
+					{
+						text: constants.DATABASE_BACKUP_SQL_VM_PAGE_BLOB_URL_LABEL,
+						url: 'https://learn.microsoft.com/en-us/sql/relational-databases/backup-restore/sql-server-backup-to-url'
```

Comment:
this is awesome! thanks for adding this so quickly. I know I didn't include these ideas on the original work item, but should we also link the known issues page that talks about this more (https://learn.microsoft.com/en-us/azure/dms/known-issues-azure-sql-migration-azure-data-studio#azure-sql-managed-instance-and-sql-server-on-azure-virtual-machine-known-issues-and-limitations)

Reply:
Good call, resolving this issue for now since we'll separate this into its own PR, but I'll be sure to update the link